commitlint/plugins.test.ts

```diff
@@ -612,7 +611,21 @@ test("footer-refs-validity6", () => {
     );
     expect(footerRefsValidity6.status).toBe(0);
 });
-*/
+
+// This test reflects this issue: https://github.com/nblockchain/conventions/issues/125
+test("footer-refs-validity7", () => {
+    let commitMsgWithWithoutFooter =
+        "foo: blah blah" +
+        "\n\n" +
+        "Otherwise we get this error in the server:" +
+        "\n\n" +
+        "```" +
+        "System.InvalidCastException:" +
+        "Jul 12 10:58:47 vultr env[2647]: at Npgsql.Internal.TypeHandlers.DateTimeHandlers.TimestampHandler.ValidateAndGetLength(DateTime value, NpgsqlParameter parameter)" +
+        "```";
+    let footerRefsValidity7 = runCommitLintOnMsg(commitMsgWithWithoutFooter);
+    expect(footerRefsValidity7.status).toBe(0);
+});
```

Comment:
The simplest testcase that can expose the bug! Like in TDD. Hope you haven't forgotten TDD ;) There's no need for that ValideAndGetLength, not even the env word

Reply:
```
let commitMsgWithWithoutFooter =
        "foo: blah blah" +
        "\n\n" +
        "Otherwise we get this error in the server:" +
        "\n\n" +
        "```" +
        "[2647]: at someFunction()" +
        "```";
```

Reply:
why 2647 and not 1? why at someFunction()? You don't need those to expose the bug.

Reply:
You understand the bug so keep seeking for the simplest testcase.

Reply:
it could be `[1] foo` inside the code block too but I thought something similar to the original error message could be more understandable

Reply:
yeah, why not?

Reply:
done